README.md

```diff
@@ -1,2 +1,167 @@
+# Explainer for the TODO API
+
+**Instructions for the explainer author: Search for "todo" in this repository and update all the
+instances as appropriate. For the instances in `index.bs`, update the repository name, but you can
+leave the rest until you start the specification. Then delete this block of text.**
+
+This proposal is an early design sketch by [TODO: team] to describe the problem below and solicit
+feedback on the proposed solution. It has not been approved to ship in Chrome.
+
+## Proponents
+
+- [Proponent team 1]
+- [Proponent team 2]
+- [etc.]
+
+## Participate
+- [Issue tracker]
+- [Discussion forum]
+
+## Table of Contents [if the explainer is longer than one printed page]
+
+<!-- Update this table of contents by running `npx doctoc README.md` -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Goals [or Motivating Use Cases, or Scenarios]](#goals-or-motivating-use-cases-or-scenarios)
+- [Non-goals](#non-goals)
+- [User research](#user-research)
+- [[API 1]](#api-1)
+- [[API 2]](#api-2)
+- [Key scenarios](#key-scenarios)
+  - [Scenario 1](#scenario-1)
+  - [Scenario 2](#scenario-2)
+- [Detailed design discussion](#detailed-design-discussion)
+  - [[Tricky design choice #1]](#tricky-design-choice-1)
+  - [[Tricky design choice 2]](#tricky-design-choice-2)
+- [Considered alternatives](#considered-alternatives)
+  - [[Alternative 1]](#alternative-1)
+  - [[Alternative 2]](#alternative-2)
+- [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
+- [References & acknowledgements](#references--acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+TODO: Fill this in using https://tag.w3.org/explainers/ as a reference.
+
+[The "executive summary" or "abstract".
+Explain in a few sentences what the goals of the project are,
+and a brief overview of how the solution works.
+This should be no more than 1-2 paragraphs.]
+
+## Goals [or Motivating Use Cases, or Scenarios]
+
+[What is the **end-user need** which this project aims to address?]
+
+## Non-goals
+
+[If there are "adjacent" goals which may appear to be in scope but aren't,
+enumerate them here. This section may be fleshed out as your design progresses and you encounter necessary technical and other trade-offs.]
+
+## User research
+
+[If any user research has been conducted to inform the design choices presented
+discuss the process and findings.
+We strongly encourage that API designers consider conducting user research to
+verify that their designs meet user needs and iterate on them,
+though we understand this is not always feasible.]
+
+<!-- In your initial explainer, you shouldn't be attached or appear attached to any of the potential
+solutions you describe below this. -->
+
+## [Potential Solution 1]
```

Comment:
The potential solution(s) should be below the "tricky design choice" section below, but DEFINITELY after the Key Scenarios.